CHANGELOG/CHANGELOG-0.4.md

```diff
@@ -0,0 +1,190 @@
+# Release notes for v0.4.0
+
+[Documentation](https://docs.k8s.io/docs/home)
```

Comment:
I think documentation should point at https://kubernetes-csi.github.io/

Reply:
Fixed.